### PR TITLE
Port stable diffision guide to Keras 3

### PR DESCRIPTION
**Pull request by tirthasheshpatel**
This PR ports the KerasCV Stable Diffusion guide to Keras 3.

@mattdangerw Would you mind running the autogen script to generate the ipynb and md files? I ran the notebook on Colab on an A100 GPU and everything works. Note that a GPU with at least 40GB memory is required for the mixed precision model to compile. Other GPUs with less memory throw an OOM error.

### CODE REVIEW

File: guides/ipynb/keras_cv/generate_images_with_stable_diffusion.ipynb [hunk 1]
```diff
@@ -37,29 +37,36 @@
     "these powerful performance boosts, and explore the performance benefits\n",
     "that they offer.\n",
     "\n",
+    "**Note:** To run this guide on the `torch` backend, please set `jit_compile=False`\n",
+    "everywhere. XLA compilation for StableDiffusion does not currently work with\n",
```
Comment (pcoet) [inline guides/ipynb/keras_cv/generate_images_with_stable_diffusion.ipynb:41]: It seems like the AI community typically uses two words, "Stable Diffusion". Let's follow that convention, unless we're talking about a class or other code.

File: guides/keras_cv/generate_images_with_stable_diffusion.py [hunk 2]
```diff
@@ -25,16 +25,21 @@
 these powerful performance boosts, and explore the performance benefits
 that they offer.
 
+**Note:** To run this guide on the `torch` backend, please set `jit_compile=False`
+everywhere. XLA compilation for StableDiffusion does not currently work with
```
Comment (pcoet) [inline guides/keras_cv/generate_images_with_stable_diffusion.py:29]: "StableDiffusion" -> "Stable Diffusion" OR "StableDiffusion" -> `StableDiffusion`

File: guides/md/keras_cv/generate_images_with_stable_diffusion.md [hunk 3]
```diff
@@ -28,19 +28,32 @@ In this guide, we will explore KerasCV's Stable Diffusion implementation, show h
 these powerful performance boosts, and explore the performance benefits
 that they offer.
 
+**Note:** To run this guide on the `torch` backend, please set `jit_compile=False`
+everywhere. XLA compilation for StableDiffusion does not currently work with
```
Comment (pcoet) [inline guides/md/keras_cv/generate_images_with_stable_diffusion.md:32]: See previous comments about "StableDiffusion" usage.

**Review by pcoet — APPROVED** [review]
LGTM. Just called out one nitty usage thing...

**Review by fchollet — COMMENTED** [review]
Thanks for the PR!

File: guides/md/keras_cv/generate_images_with_stable_diffusion.md [hunk 4]
```diff
@@ -279,18 +300,11 @@ Enabling mixed precision computation in Keras
...
-Your GPU will likely run quickly with dtype policy mixed_float16 as it has compute capability of at least 7.0. Your GPU: NVIDIA A100-SXM4-40GB, compute capability 8.0
-
-```
-</div>
 That's all.  Out of the box - it just works.
 
 
 ```python
-model = keras_cv.models.StableDiffusion()
+model = keras_cv.models.StableDiffusion(jit_compile=False)
```
[the remark this answers is not in the document]
Comment (tirthasheshpatel) [reply guides/md/keras_cv/generate_images_with_stable_diffusion.md:307]: The guide focuses on benchmarking the stable diffusion model with and without `jit_compile`. Hence `jit_compile=False` initially in the guide. Later, performance benifits of `jit_compile=True` are introduced.

[the remark this answers is not in the document]
Comment (mattdangerw) [reply guides/md/keras_cv/generate_images_with_stable_diffusion.md:307]: Yeah, honestly I am just doing this to match the copy of the guide for now. The guides turns on `jit_compile=True` later as a performance improvement, which makes no sense if we have been running with XLA the whole time. We should re-write the copy to make more sense in the Keras 3 world.

**Comment by mattdangerw** [comment]
ok, comments addressed!

File: guides/md/keras_cv/generate_images_with_stable_diffusion.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (fchollet) [reply guides/md/keras_cv/generate_images_with_stable_diffusion.md:301]: Got it!

**Review by fchollet — APPROVED** [review]
LGTM, thank you!

**Pull request merged**